src/Bicep.Core.RegistryClient/DownloadManifestResult.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.IO;
+
+namespace Bicep.Core.RegistryClient
+{
+    public class DownloadManifestResult
+    {
+        internal DownloadManifestResult(Stream content)
```

Comment:
@annelo-msft I found myself implementing an in-memory mock of all the operations we're using and had to make this public to make to make it work.